lint.js

```
@@ -1,3 +1,6 @@
 const add = function(a, b) {
     return a + b;
 }
+const add2 = function(a, b) {
```

ESLint

[error] 'add2' is assigned a value but never used.

(no-unused-vars)